modules/resource-graph/query/.test/main.test.bicep

```diff
@@ -0,0 +1,15 @@
+targetScope = 'subscription'
+
+// ========== //
+// Test Cases //
+// ========== //
+
+var namePrefix = 'crml'
+
+// TEST 1 - COMMON
```

Comment:
While it seams over the top, please let's also a min test.... Which I guess will be very small 😄

Reply:
Min test added.